docs/source/processing.rst

```diff
@@ -491,4 +491,53 @@ Both :obj:`datasets.Dataset` and :obj:`datasets.DatasetDict` objects can be save
 Controling the cache behavior
 -----------------------------------
 
-[UNDER CONSTRUCTION]
+When applying transforms on a dataset, the data are stored in cache files.
+The caching mechanism allows to reload an existing cache file if it's already been computed.
+
+Reloading a dataset is possible since the cache files are named using the dataset fingerprint, which is updated after each transform.
```

Comment:
Note that the caching extend beyond sessions. Re-running the very same dataset processing methods (in the same order and on the same data files) in a different session will load from the same cache files.

This is possible thanks to a custom hashing function that works with most python objects (see fingerprinting section below).